### DISCUSSION TITLE
Two max7219

### DISCUSSION THREAD

**Discussion by pableitordasauto** [shanteacontrols/OpenDeck#273] [General]
Hi, 
Is it possible make and configure a board with two max7219 to control more than 64 leds?
Thanks!!

**Comment by paradajz** [comment]
It is now.
https://github.com/shanteacontrols/OpenDeck/commit/23d037c3af654ea75098ec1da7d843c16efbefff
https://github.com/shanteacontrols/OpenDeck/wiki/Creating-custom-board-variant#max7219

**Reply by pableitordasauto** [reply]
SUPER!!!
thanks!!!
